packages/edit-site/src/components/editor/index.js

```diff
@@ -138,9 +138,20 @@ export default function Editor() {
 	);
 	const isReady = editedPostType !== undefined && editedPostId !== undefined;
 
+	let title;
+	if ( isReady && editedPost ) {
+		const type =
+			editedPostType === 'wp_template'
+				? __( 'Template' )
+				: __( 'Template Part' );
+		title = `${ editedPost.title?.rendered } ‹ ${ type } ‹ ${ __(
+			'Editor (beta)'
```

Comment:
I think we should remove "beta" now.

Reply:
Also, I'm not entirely certain that we should concat translated items like that, we should probably be using `sprintf` here.

Reply:
There's also a hook `useEditedEntityRecord` that would save a few lines of code here (allows to call `getTitle` directly and also provide access to the `record` if needed.

Reply:
I have added a [follow up PR here](https://github.com/WordPress/gutenberg/pull/48053) to cover off these suggestions.